rapido.css

```diff
@@ -47,8 +47,8 @@ article.rapido {
 /* aside */
 
 .rapido section > aside {
-	width: 100%;
-	max-width: 600px;
+	width: 600px;
```

Comment:
Authors can change the `width` with inline style. The `max-width` property ensure the right width on smaller screens.